be/src/olap/tablet_manager.cpp

```diff
@@ -1307,6 +1264,18 @@ void TabletManager::unregister_clone_tablet(int64_t tablet_id) {
     shard.tablets_under_clone.erase(tablet_id);
 }
 
+std::pair<std::map<doris::TabletSchemaPath, std::string>::iterator, bool> TabletManager::register_transition_tablet(int64_t tablet_id, TSchemaHash schema_hash, size_t path_hash, std::string reason) {
```

Comment:
return type is Status,  when register fail, maybe return Status::Error("tablet xxx is doing yyy")